### DISCUSSION TITLE
Game refuses to compile

### DISCUSSION THREAD

**Discussion by BigOlDummyhead** [FunkinCrew/Funkin#1715] [Compiling help] [answered]
I have gotten it to work before. It was doing fine like 2 weeks ago. I do not understand what has changed and idk what to do
![d7ca74af8c7205d17bd1eaf27bd1fa14](https://user-images.githubusercontent.com/89865965/131549285-3857fee8-9109-42cf-8226-a9c16bb48bd3.jpg)
<img width="289" alt="da880c611ed930e817978d87414a684a" src="https://user-images.githubusercontent.com/89865965/131549301-e68c5f73-1f07-4ee9-8e57-05b07a33ca55.png">
these are my errors

**Comment by ninjamuffin99** [comment]
you can probably just delete the Snd.hx file

**Reply by steviegt6** [reply]
Curious: what was it for in the first place?

**Reply by BigOlDummyhead** [reply]
No dice :/ It always loads a bunch of stuff and then just stops...
Here is the log

`> Executing task: lime build windows -debug --haxelib=hxcpp-debug-server --connect 6000 <

C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access

Compiling group: haxe
cl.exe -Iinclude -IC:/HaxeToolkit/haxe/lib/discord_rpc/git/linc/ -nologo /WX- /fp:precise -DHX_WINDOWS -GR -Od(optim-std) -Zi(optim-std) -FdC:\Users\(Me)\Desktop\Funkin-master\export\debug\windows\obj\obj/msvc1964-debug-nc/vc.pdb(optim-std) -Zi(debug) -FdC:\Users\(Me)\Desktop\Funkin-master\export\debug\windows\obj\obj/msvc1964-debug-nc/vc.pdb(debug) -Od(debug) -O2(release) -bigobj -Os(optim-size) -FS -Oy- -c 
-EHs -GS- -IC:/HaxeToolkit/haxe/lib/hxcpp/4,2,1/include -DHXCPP_DEBUG -DHXCPP_M64 -DHXCPP_VISIT_ALLOCS(haxe) -DHXCPP_DEBUGGER(haxe) -DHX_SMART_STRINGS(haxe) -DHXCPP_API_LEVEL=400(haxe) -D_CRT_SECURE_NO_DEPRECATE -D_ALLOW_MSC_VER_MISMATCH -D_ALLOW_ITERATOR_DEBUG_LEVEL_MISMATCH -wd4996 ... tags=[haxe,static]
 - src/haxe/io/Input.cpp
 - src/io/newgrounds/objects/events/_Response/Response_Impl_.cpp
 - src/sys/io/FileOutput.cpp
 - src/__boot__.cpp
 - src/__files__.cpp
 - src/sys/io/_Process/Stdin.cpp
 - src/sys/io/_Process/Stdout.cpp
 - src/hxcpp/debug/jsonrpc/eval/Parser.cpp 
 - src/hxcpp/debug/jsonrpc/VariablesPrinter.cpp 
 - src/lime/system/CFFI.cpp  [haxe,release]
 - src/lime/system/System.cpp  [haxe,release]
 - src/lime/utils/AssetCache.cpp  [haxe,release]
 - src/flixel/addons/ui/U.cpp 
 - src/openfl/net/SharedObject.cpp  [haxe,release]
 - src/polymod/ModMetadata.cpp 
 - src/flixel/addons/ui/FlxUI.cpp 
 - src/flixel/system/debug/console/Console.cpp 
 - src/flixel/util/FlxSave.cpp 
 - src/openfl/display/Stage.cpp  [haxe,release]
 - src/flixel/addons/ui/FlxUIState.cpp 
 - src/__resources__.cpp
 - src/polymod/format/CSVParseFormat.cpp 
 - src/sys/net/Host.cpp 
 - src/hxcpp/debug/jsonrpc/Server.cpp 
 - src/sys/net/Socket.cpp 
 - src/polymod/format/XMLParseFormat.cpp 
 - src/sys/Http.cpp 
 - src/ApplicationMain.cpp 
 - src/polymod/format/JSONParseFormat.cpp 
 - src/lime/app/_Future/FutureWork.cpp  [haxe,release]
 - src/sys/ssl/_Socket/SocketOutput.cpp 
 - src/hscript/Interp.cpp 
 - src/lime/ui/FileDialog.cpp  [haxe,release]
 - src/sys/io/FileInput.cpp 
 - src/sys/io/Process.cpp 
 - src/hxcpp/debug/jsonrpc/eval/Interp.cpp 
 - src/sys/ssl/Socket.cpp 
 - src/openfl/display/Window.cpp  [haxe,release]
 - src/cpp/Lib.cpp 
 - src/lime/app/Future.cpp  [haxe,release]
 - src/polymod/format/JsonHelp.cpp 
 - src/sys/net/_Socket/SocketOutput.cpp 
 - src/flixel/system/debug/watch/EditableTextField.cpp 
 - src/haxe/ds/BalancedTree.cpp 
 - src/sys/ssl/_Socket/SocketInput.cpp 
 - src/openfl/text/TextField.cpp  [haxe,release]
 - src/polymod/Polymod.cpp 
 - src/polymod/backends/PolymodAssetLibrary.cpp 
 - src/flixel/system/debug/completion/CompletionHandler.cpp 
 - src/EReg.cpp 
 - src/sys/net/_Socket/SocketInput.cpp 

Compiling group: __resources__
cl.exe -nologo /WX- /fp:precise -DHX_WINDOWS -GR -Od(optim-std) -Zi(optim-std) -FdC:\Users\(Me)\Desktop\Funkin-master\export\debug\windows\obj\obj/msvc1964-debug-nc/vc.pdb(optim-std) -Zi(debug) -FdC:\Users\(Me)\Desktop\Funkin-master\export\debug\windows\obj\obj/msvc1964-debug-nc/vc.pdb(debug) -Od(debug) -O2(release) -bigobj -Os(optim-size) -FS -Oy- -c -EHs -GS- -IC:/HaxeToolkit/haxe/lib/hxcpp/4,2,1/include -DHXCPP_DEBUG -DHXCPP_M64 -DHXCPP_VISIT_ALLOCS(haxe) -DHXCPP_DEBUGGER(haxe) -DHX_SMART_STRINGS(haxe) -DHXCPP_API_LEVEL=400(haxe) -D_CRT_SECURE_NO_DEPRECATE -D_ALLOW_MSC_VER_MISMATCH -D_ALLOW_ITERATOR_DEBUG_LEVEL_MISMATCH -wd4996 ... tags=[haxe]
 - src/resources/__res_20.cpp  [optim-none]
 - src/resources/__res_81.cpp  [optim-none]
 - src/resources/__res_22.cpp  [optim-none]
 - src/resources/__res_10.cpp  [optim-none]
 - src/resources/__res_77.cpp  [optim-none]
 - src/resources/__res_12.cpp  [optim-none]
 - src/resources/__res_30.cpp  [optim-none]
 - src/resources/__res_79.cpp  [optim-none]
 - src/resources/__res_69.cpp  [optim-none]
 - src/resources/__res_67.cpp  [optim-none]
 - src/resources/__res_73.cpp  [optim-none]
 - src/resources/__res_51.cpp  [optim-none]
 - src/resources/__res_59.cpp  [optim-none]
 - src/resources/__res_9.cpp  [optim-none]
 - src/resources/__res_49.cpp  [optim-none]
 - src/resources/__res_31.cpp  [optim-none]
 - src/resources/__res_33.cpp  [optim-none]
 - src/resources/__res_45.cpp  [optim-none]
 - src/resources/__res_35.cpp  [optim-none]
 - src/resources/__res_6.cpp  [optim-none]
 - src/resources/__res_74.cpp  [optim-none]
 - src/resources/__res_76.cpp  [optim-none]
 - src/resources/__res_52.cpp  [optim-none]
 - src/resources/__res_4.cpp  [optim-none]
 - src/resources/__res_50.cpp  [optim-none]
 - src/resources/__res_58.cpp  [optim-none]
 - src/resources/__res_44.cpp  [optim-none]
 - src/resources/__res_34.cpp  [optim-none]
 - src/resources/__res_36.cpp  [optim-none]
Link: ApplicationMain-debug.exe
   Creating library ApplicationMain-debug.lib and object ApplicationMain-debug.exp

Terminal will be reused by tasks, press any key to close it.`

Was wondering if i had files missing? So i just redownloaded the game files. Now I am getting an error saying "haxe.Utf8 is deprecated. Use UnicodeString instead." won't boot and and am still getting nowhere

**Comment by BigOlDummyhead** [comment, marked as answer]
neeeveeermiiiind you can ignore me. I used the command prompt instead of visual studio code to build the game and that worked. A little annoying, cuz i was following someone's tutorial and had compiled the game with that before so i don't know what changed, but at least i got it running again (_ _ ;)
